templates/publisher/settings.html

```diff
@@ -122,102 +120,100 @@
+                <div class="js-multiselect-holder{% if blacklist_country_codes %} is-disabled{% endif %}"></div>
+              </div>
+              <div class="js-blacklist p-multiselect__wrapper">
+                <input type="radio" name="territories_custom_type" value="blacklist" id="territories_custom_blacklist"
+                       {% if blacklist_country_codes %}
+                       checked="checked"
+                       {% endif %}
+                       />
+                <label for="territories_custom_blacklist">Exclude these territories</label>
+                <input type="text" class="js-multiselect-input" name="blacklist_countries" value="
+                                                                                                  {% for territory in blacklist_country_codes %}
+                                                                                                  {{ territory }}{{ "," if not loop.last }}
+                                                                                                                       {% endfor %}" />
+                <div class="js-multiselect-holder{% if whitelist_country_codes or not blacklist_country_codes %} is-disabled{% endif %}"></div>
+              </div>
             </div>
           </div>
         </div>
       </div>
 
-      <div class="p-strip is-shallow">
-        <div class="row">
-          <hr class="u-no-margin--bottom" />
-        </div>
+      <div class="row p-form__group">
+        <hr class="u-no-margin--bottom" />
       </div>
 
-      <div class="row">
+      <div class="row p-form__group">
         <div class="col-2">
-          Collaboration:
+          <label class="p-form__label">Collaboration:</label>
         </div>
         <div class="col-8">
-          <a class="p-link--external" href="https://dashboard.snapcraft.io/snaps/{{snap_name}}/collaboration/">Manage collaborators in dashboard.snapcraft.io</a>
+          <div class="p-form__control">
+            <a class="p-link--external" href="https://dashboard.snapcraft.io/snaps/{{snap_name}}/collaboration/">Manage collaborators in dashboard.snapcraft.io</a>
+          </div>
         </div>
       </div>
 
-      <div class="p-strip is-shallow">
-        <div class="row">
-          <hr class="u-no-margin--bottom" />
-        </div>
+      <div class="row p-form__group">
+        <hr class="u-no-margin--bottom" />
       </div>
 
-      <div class="row">
+      <div class="row p-form__group">
         <div class="col-2">
-          Store:
+          <label class="p-form__label">Store:</label>
         </div>
         <div class="col-8">
-          {{ store }}
+          <div class="p-form__control">
+            {{ store }}
+          </div>
         </div>
       </div>
 
-      <div class="p-strip is-shallow">
-        <div class="row">
-          <hr class="u-no-margin--bottom" />
-        </div>
+      <div class="row p-form__group">
+        <hr class="u-no-margin--bottom" />
```

Comment:
I wish there was a nicer way to do these instead of wrapping hr into rows or strips... But let's not overthink it for now.

Reply:
@bartaz @Lukewh @therealjuan do we even need the dividing lines between the listing sections?

Reply:
I removed them, but it looked a bit odd as they're not really related

Reply:
@Lukewh Ah ok makes sense, just thought I'd ask in case there wasn't a reason for it 😉